fontique/src/backend/coretext.rs

```diff
@@ -75,97 +71,45 @@ impl SystemFonts {
     pub(crate) fn fallback(&mut self, key: impl Into<FallbackKey>) -> Option<FamilyId> {
         let key = key.into();
         let sample = key.script().sample()?;
-        self.fallback_for_text(sample, key.locale(), false)
+        let font = fallback_font_for_text(sample, key.locale(), false)?;
+        let family_name = unsafe { CTFontCopyFamilyName(&font) };
+        self.name_map.get(&family_name.to_string()).map(|n| n.id())
     }
 }
 
-impl SystemFonts {
-    fn fallback_for_text(
-        &mut self,
-        text: &str,
-        locale: Option<&str>,
-        prefer_ui: bool,
-    ) -> Option<FamilyId> {
-        let font = fallback_for_text(text, locale, prefer_ui)?;
-        self.name_map.get(&font.family_name()).map(|n| n.id())
+fn get_base_font(prefer_ui_font: bool) -> CFRetained<CTFont> {
```

Comment:
I don't massively like this `get_` prefix, but I don't think just `base_font` is great either. The methods we're calling all use 'Create'...

This isn't a blocking concern

Reply:
Renamed `get_base_font` and `fallback_font_for_text` to both have `create_` prefixes.